neqo-transport/src/connection/tests/stream.rs

```diff
@@ -549,3 +551,75 @@ fn no_dupdata_readable_events_empty_last_frame() {
     let _ = server.process(out_second_data_frame.dgram(), now());
     assert!(!server.events().any(stream_readable));
 }
+
+fn change_flow_control(stream_type: StreamType, new_fc: u64) {
+    const RECV_BUFFER_START: u64 = 3000;
```

Comment:
This is a little easier if you start with a value that is smaller than a single packet.  That way you don't have to worry about exchanging more than one packet when you write something.